mergify_engine/mergify_context.py

```diff
@@ -119,7 +105,8 @@ def _get_valid_users(self):
         ]
         return bots + valid_collabs
 
-    def _get_consolidated_reviews(self):
+    @functools_bp.cached_property
```

Comment:
are you sure you want a property knowing it should be async in the future?

Reply:
Yes, it should be cached, when I will render it async, I just have to grep all `functools_bp.cached_property` ;)